js/tests.js

```diff
@@ -52,4 +52,47 @@ describe("get auto-complete context", function() {
         expect(parser.autoCompleteContext('Hi @(CONCAT("@con')).toBeNull();
         expect(parser.autoCompleteContext('Hi @("!" & "@con')).toBeNull();
     });
+
+    it("ignore parenthesis triggering functions completions for variables", function() {
+        expect(parser.autoCompleteContext("Hi @(contact.age")).toBe('contact.age');
+    });
+
+    it('ignore the parenthesis triggering functions completions for functions', function() {
+        expect(parser.autoCompleteContext("Hi @(SUM")).toBe('SUM');
+    });
+
+    it("matches the function without parameters", function() {
+        expect(parser.autoCompleteContext("Hi @(SUM(")).toBe('SUM');
```

Comment:
I thought we agreed yesterday that that wasn't the desired behaviour? Here's another example of what I think is the desired behaviour...

`@(RAND` - display search results for "RAND" (2 results: RAND, RANDBETWEEN)
`@(RAND(` - display summary box for "RAND"
`@(RAND(c` - display search results for "c"

Ideally we'd also switch back to the summary box between parameters, e.g.

`@(RAND(contact.age` - display search results for "contact.age"
`@(RAND(contact.age,` - display summary box for "RAND"
`@(RAND(contact.age, f` - display search results for "f"

Summary box of course doesn't have to be a separate thing - can be restyling of the usual at.js search results dropdown.